packages/app-cli/app/services/plugins/PluginRunner.ts

```diff
@@ -90,6 +110,13 @@ export default class PluginRunner extends BasePluginRunner {
 		});
 	}
 
+	public async stop(plugin: Plugin): Promise<void> {
+		// TODO: Node VM doesn't support stopping plugins without running them in a child process.
+		// For now, we stop the plugin by making interactions with the Joplin API throw Errors.
+		const proxy = this.sandboxProxies.get(plugin.id);
+		proxy?.stop();
```

Comment:
This is used to test the plugin unloading logic. An alternative to this could be to mock `.stop` when running in a testing environment (e.g. with `jest.spyOn`).